packages/@rnw-scripts/jest-unittest-config/jest.unittest.config.js

```diff
@@ -22,9 +22,6 @@ module.exports = {
   // The pattern or patterns Jest uses to detect test files
   testRegex: '/test/.*\\.test\.ts$',
 
-  // Specifies the maximum number of workers the worker-pool will spawn for running tests.
-  maxWorkers: 1,
-
```

Comment:
Unrelated, but this shouldn't have been there...

Reply:
It looks like reenabling JS UT parallelism leads to failiure of one of our tests. Going to leave fixing the test failure that slipped in for later.
```
ERR!  FAIL  src/test/sanitize.test.ts
ERR!   √ Sanitize message, no-op (2 ms)
ERR!   √ Sanitize message, project_dir (1 ms)
ERR!   √ Sanitize message, node_modules (1 ms)
ERR!   √ Sanitize message, other path
ERR!   √ Sanitize message, forward slashes (1 ms)
ERR!   √ Sanitize message, file share path
ERR!   √ Sanitize message, with cpu/thread id
ERR!   √ Error code (1 ms)
ERR!   √ Sanitize stack frame (1 ms)
ERR!   √ basic setup (2 ms)
ERR!   × thrown exception a->b, hello world (39 ms)
ERR!   √ throw exception with error code (19 ms)
ERR!   × thrown exception a->b, hello path (8 ms)
ERR!   √ trackEvent should not identify roleInstance (1 ms)
ERR!   √ No message (1 ms)
ERR! 
ERR!   ● thrown exception a->b, hello world
ERR! 
ERR!     expect(received).toBeTruthy()
ERR! 
ERR!     Received: false
ERR! 
ERR!       262 |   Telemetry.client!.flush();
ERR!       263 |
ERR!     > 264 |   expect(pass).toBeTruthy();
ERR!           |                ^
ERR!       265 |   Telemetry.client!.clearTelemetryProcessors();
ERR!       266 |   Telemetry.client!.addTelemetryProcessor(sanitizeEnvelope);
ERR!       267 |   done();
ERR! 
ERR!       at Object.<anonymous> (src/test/sanitize.test.ts:264:16)

```